changelogs/changelog.yaml

```diff
@@ -0,0 +1,108 @@
+ancestor: null
+releases:
+  1.0.0:
+    modules:
+    - description: ACL interfaces resource module
+      name: ios_acl_interfaces
+      namespace: ''
+    - description: ACLs resource module
+      name: ios_acls
+      namespace: ''
+    - description: Manage multiline banners on Cisco IOS devices
+      name: ios_banner
+      namespace: ''
+    - description: Configure global BGP protocol settings on Cisco IOS.
+      name: ios_bgp
+      namespace: ''
+    - description: Run commands on remote devices running Cisco IOS
+      name: ios_command
+      namespace: ''
+    - description: Manage Cisco IOS configuration sections
+      name: ios_config
+      namespace: ''
+    - description: Collect facts from remote devices running Cisco IOS
+      name: ios_facts
+      namespace: ''
+    - description: (deprecated, removed after 2022-06-01) Manage Interface on Cisco
+        IOS network devices
+      name: ios_interface
+      namespace: ''
+    - description: Interfaces resource module
+      name: ios_interfaces
+      namespace: ''
+    - description: (deprecated, removed after 2022-06-01) Manage Layer-2 interface
+        on Cisco IOS devices.
+      name: ios_l2_interface
+      namespace: ''
+    - description: L2 interfaces resource module
+      name: ios_l2_interfaces
+      namespace: ''
```

Comment:
is there a specific need to keep the namespace, if we're not populating the same?

Reply:
Yes, it is for the nested folder in the modules directory for eg the nxos collection has modules under the storage directory. If the nested folder is not present the changelog tool expects namespace value to `''`